css/dark-mode.css

```
@@ -217,3 +217,8 @@ html.dark-mode .x1ybostu {
 html.dark-mode .x4noic3 {
 	background-color: var(--secondary-button-background);
 }
+
+/* Invite notification: join room */
+html.dark-mode .x1qhmfi1 {
+	background-color: var(--container-color);
```

What about secondary button background color? It might look better in this case. The darker button on a lighter widget which is itself on a darker background seems kind of out of place.

```suggestion
	background-color: var(--secondary-button-background);
```